### PR TITLE
Add ignore_suffix Config

### PR DESCRIPTION
**Pull request by alonelaval**

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please create a new branch from the upstream master. The commit loga include many unrelated things

**Comment by alonelaval** [comment]
> Please create a new branch from the upstream master. The commit loga include many unrelated things

ok, next i will create new branch...

File: skywalking/trace/context/__init__.py [hunk 1]
```diff
@@ -76,6 +88,15 @@ def new_exit_span(self, op: str, peer: str, carrier: 'Carrier' = None) -> Span:
 
         return span
 
+    def ignore_check(self, op: str, kind: Kind):
+        suffix_idx = op.rfind(".")
+        if suffix_idx > -1 and config.ignore_suffix.find(op[op.rfind("."):]) > -1:
```
Comment (kezhenxu94) [inline skywalking/trace/context/__init__.py:93]:
```suggestion
        if suffix_idx > -1 and config.ignore_suffix.find(op[suffix_idx:]) > -1:
```

**Comment by alonelaval** [comment]
> LGTM

I tested in 8.0.1 backend...running will

**Pull request merged**